tests/subworkflows/ebi-metagenomics/reads_qc/nextflow.config

```diff
@@ -0,0 +1,14 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: SEQTK_SEQ {
```

Comment:
This one should be also included in a nextflow.config file in the subworkflow root folder as https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/bam_docounts_contamination_angsd/nextflow.config

Reply:
I don't think so.. it's just for the key "-a" parameter